assets/base.css

```diff
@@ -209,15 +209,15 @@ html.no-js .no-js-hidden {
 }
 
 .element-margin {
-  margin-top: 5rem;
+  margin-top: calc(5rem + var(--page-width-margin));
 }
 
 .spaced-section {
-  margin-top: 5rem;
+  margin-top: calc(5rem + var(--page-width-margin));
 }
 
 .spaced-section:last-child {
-  margin-bottom: 5rem;
+  margin-bottom: calc(5rem + var(--page-width-margin));
 }
```

Comment:
Something to keep in mind is that we will need to

1) Go over all files and see where the spacing is actually being added through the padding and update it

2) Go over all files and see where we need to adjust the negative margin: https://github.com/Shopify/dawn/blob/b85b3f3971cc83a4b4bb56be93145b8555ee8605/assets/component-product-grid.css#L5-L21

Video: https://screenshot.click/08-55-665y0-g62o6.mp4